rpc/nns/names.go

```diff
@@ -0,0 +1,13 @@
+package nns
+
+// A set of standard contract names deployed into NeoFS sidechain.
+const (
+	NameAlphabet   = "alphabet"
```

Comment:
Alphabet contracts are currently special cuz they are named with indices `alphabet0`, `alphabet1`, etc.

Reply:
Maybe it should be `NameAlphabetPrefix` then?

Reply:
yeah, such name gonna be much more correct

Reply:
Fixed.